src/actor.cpp

```diff
@@ -198,21 +198,23 @@ actor_t::prototype() const {
 void
 actor_t::run() {
     m_acceptor.apply([this](std::unique_ptr<tcp::acceptor>& ptr) {
+        tcp::endpoint endpoint;
+
         try {
-            ptr = std::make_unique<tcp::acceptor>(*m_asio, tcp::endpoint {
+            endpoint = tcp::endpoint {
                 m_context.config.network.endpoint,
                 m_context.mapper.assign(m_prototype->name())
-            });
+            };
+
+            ptr = std::make_unique<tcp::acceptor>(*m_asio, endpoint);
         } catch(const std::system_error& e) {
-            COCAINE_LOG_ERROR(m_log, "unable to bind local endpoint for service: %s",
-                error::to_string(e));
+            COCAINE_LOG_ERROR(m_log, "unable to bind local endpoint %s for service: %s",
+                endpoint, error::to_string(e));
```

Comment:
This actually is going to be a little bit weird error message in case the exception comes from the Port Mapper, something like: «unable to bind local endpoint 0.0.0.0:0 for service: named port is already in use», but I guess it's fine.